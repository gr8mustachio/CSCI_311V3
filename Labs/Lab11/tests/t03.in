6 10
0 3
0 4
3 2
3 4
1 2
1 5
2 5
4 1
4 2
4 5
bfs 0
bfs 3
bfs 4
