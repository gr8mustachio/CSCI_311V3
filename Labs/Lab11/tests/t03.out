Graph is:
0: 3 4 
1: 2 5 
2: 5 
3: 2 4 
4: 1 2 5 
5: 
Running BFS on the source 0:
0 3 4 2 1 5 
Running BFS on the source 3:
3 2 4 5 1 
Running BFS on the source 4:
4 1 2 5 
